operations/elections.rst

```diff
@@ -0,0 +1,78 @@
+
+.. note::
+    Přečíst ještě `PEP 8102 <https://www.python.org/dev/peps/pep-8102/>`__ a případně se inspirovat? Dořešit.
+
+Kdy se volí
+-----------
+
+- Předchozí volba: **8.4.2019**
+- Příští volba: **?.?.2022**, předpokládá se termín březen/duben
+- Příprava pro rok 2022 začíná: od 1. ledna
+
+Příprava
+--------
+
+- Postupuje se podle stanov, svoláváme :ref:`členskou schůzi <clenska-schuze>`.
+- Výbor na nějaké schůzi s předstihem určí přesné datum členské schůze.
+- Někdo z výboru vezme e-mailové adresy členů ze `seznamu členů <https://docs.google.com/spreadsheets/d/1n8hzBnwZ5ANkUCvwEy8rWsXlqeAAwu-5JBodT5OJx_I/edit#gid=0>`__ a pošle jim pozvánku na členskou schůzi. Pozvánka musí obsahovat místo, čas a program jednání členské schůze. Kromě toho by měl výbor schůzi oznámit i v kanálu :slack:`#pyvec-members`. (Pro inspiraci, pozvánka na členskou schůzi `EuroPython Society <https://www.europython-society.org/europython-society-general-assembly-2020/>`__.)
+- Výbor dá členům vědět, jak lze podávat nominace, jak kandidovat. Kandidovat do výboru může jakýkoliv člen Pyvce.
+- Ideálně se pokusí členy motivovat k tomu, aby kandidovali, a to komunikací s členy, ukazováním toho, jak výbor pracuje, jak vypadá běžná schůze výboru (např. zveřejněním video nahrávky), apod. Výbor může také neformálně oslovovat vhodné kandidáty a přesvědčovat je, že mají kandidovat.
+- Přihlášky se oznamují v kanálu :slack:`#pyvec-members`. (Pro inspiraci, `kandidatury do EuroPython Society <https://www.europython-society.org/list-of-eps-board-candidates-for-20192020/>`__.) Seznam kandidátů by bylo dobré zapisovat třeba sem do této stránky, ať je někde přehledný seznam. Pozor na GDPR, kontrolovat v `seznamu členů <https://docs.google.com/spreadsheets/d/1n8hzBnwZ5ANkUCvwEy8rWsXlqeAAwu-5JBodT5OJx_I/edit#gid=0>`__, zda daná osoba může být někde veřejně takto prezentovaná.
+- Neomezujeme kandidáty na ty přihlášené předem. Kandidáti se mohou navrhnout nebo nominovat jiné kandidáty klidně až v průběhu hlasování. Je ale pro hlasující lepší, když se mohou s kandidáty seznámit předem.
+
+Začátek voleb
+-------------
+
+- V datum schůze se udělá vykopávací meeting, koordinace probíhá přes :slack:`#pyvec-members`.
+- Výbor členy obeznámí s technickým zázemím volby.
+- Členové následně dostávají týden na to, aby asynchronně volili.
+
+.. note::
+    Jaké máme technické zázemí volby? Petr Viktorin zmiňoval, že PSF používá `Helios Voting <https://vote.heliosvoting.org/>`__. Dořešit.
+
+Mechanika voleb
+---------------
+
+- Každý člen Pyvce má 5 hlasů (ve výboru je 5 lidí).
+- Po týdnu voleb se uzavře hlasování a starý výbor vyhodnotí hlasy.
+- Bere se prvních pět, kteří dostanou nejvíc hlasů.
+- Starý výbor vyhlásí výsledky přes :slack:`#pyvec-members`.
+
+Volby při remíze
+----------------
+
+- Pokud je shoda hlasů u více kandidátů na 5. a dalším místě, volí se stejným způsobem znova mezi nimi (rekurze).
+- Každé takové nové kolo voleb trvá vždy další týden.
```

Comment:
Takze kdyz je `shoda hlasů u více kandidátů na 5. a dalším místě`, tak se musi tyden pockat, nez se hlasuje mezi tema lidma na 5. miste?

Me to prijde zbytecne, dal bych hlasovani rovnou. Vysledek bude pravdepodobne jiny, jelikoz se bude volit jen mezi danymi kandidaty, takze lidi co je nevolili vubec, nebo volili vice nez jednoho z nich budou hlasovat jinak (v teto volbe budou mit jen 1 hlas). Az pri dalsi shode (nepravdepodobne) by se mohlo pockat tyden.

Reply:
Já tam nikde nic o týden dlouhém čekání nevidím. Je tam. že další volby trvají týden a předpokládám, že začnou hned po té, co se vyhlásí výsledky předchozího kola.

Reply:
Jo takhle, ja si predstavil online meeting kde se zahlasuje, a kdyz to nevyjde, tak se ceka tyden nez se udela dalsi meeting, kde se bude hlasovat opet. Ale ted koukam ze ten meeting je jen vykopavaci a pak se hlasuje asynchrone, takze mas pravdu, diky.

Reply:
(Nemam pravo tenhle thread resolvnout.)